build.gradle

```diff
@@ -361,7 +361,8 @@ subprojects {
     // version 7.x requires Java 8
     toolVersion = '6.19'
   }
-  test.dependsOn('checkstyleMain', 'checkstyleTest')
+  // Bellemare - re-enable the below
```

Comment:
Haha yes... I will re-add the style guide checks. It currently fails on a number of things.